index.html

```diff
@@ -4,9 +4,62 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles/index.css">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <div class="container">
```

Comment:
found a few errors when I sent this through the w3 html validator. feel free to check it out here: [https://validator.w3.org/](https://validator.w3.org)